src/lib/file.js

```diff
@@ -1,9 +1,10 @@
 import { ORIGINAL_REP_NAME } from './constants';
 
-// List of Box Content API fields that the Preview SDK requires for every file. Updating this list is most likely
+// List of Box Content API fields that the Preview library requires for every file. Updating this list is most likely
 // a breaking change and should be done with care. Clients that leverage functionality dependent on this format
 // (e.g. Box.Preview.updateFileCache()) will need to be updated if this list is modified.
 const FILE_FIELDS = [
+    'id',
```

Comment:
Normally, `id` is returned with any get files API call. Now, since we also use this list for simple validation of a passed in file object, I'm explicitly adding this so developers cannot pass in a file object without an ID.